build.gradle.kts

```diff
@@ -127,7 +127,13 @@ spotless {
     indentWithSpaces(2)
     endWithNewline()
   }
+  format("xml") {
+    target("**/*.xml")
 
```

Comment:
@midgleyc - help, please.  

I'm having two issues.

A case could be made that xml in test/root/provided_data should not be changes since those are files added to the repo to be tested against.  The case could be expanded to mean nothing in test/root should be changed and, as it turns out there isn't xml in test/   There are also no xml files in src/net/sourceforge/kolmafia.  So one option would be to exclude both src/ and test/.  Alternatively just exclude test/root.  Which do you recommend and how do I change the target to implement that recommendation?

Second, I have experimented with various targets and even with I edit checkstyle.xml so that it should not pass running gradlew spotless passes and gradlew spotlessapply doesn't change anything.

Thanks.

Reply:
Added exclusions.  

checkstyle.xml  remains unresolved.

Reply:
.idea is local, not under git control and used by my IDE.  We may need a similar exclusion for users of other IDEs.